src/main/java/androidx/text/Html.kt

```diff
@@ -53,12 +58,12 @@ fun CharSequence.parseAsHtml(flags: Int? = 0): Spanned {
  * not know how to interpret.
  */
 fun CharSequence.parseAsHtml(
-    flags: Int? = 0,
+    @SuppressLint("InlinedApi") flags: Int = FROM_HTML_MODE_LEGACY,
     imageGetter: ImageGetter,
     tagHandler: TagHandler
```

Comment:
I know this was already here and isn't related to your PR, but why are these two functions? These should be marked as nullable and defaulted to null as it is documented to accept null for these.